scss/mixins/_progress.scss

```diff
@@ -9,6 +9,11 @@
     background-color: $color;
   }
 
+  // IE10+, Microsoft Edge
+  &[value] {
```

Comment:
having said all that, i found a need to start using `::-ms-fill` anyway in `scss/_progress.scss` - to suppress an unsightly right-hand border - which then led me down a whole different avenue anyway. long story short: yes, using `::-ms-fill` for this now (and now IE10+/Edge also get the rounder borders like webkit)